doc/devguide/Makefile.am

```diff
@@ -0,0 +1,59 @@
+if HAVE_SURICATA_MAN
+man1_MANS = suricata.1
+endif
```

Comment:
Anything about the man page can be removed from the devguide section.  The PDF stuff can stay though.